### PR TITLE
Update controls dependency to include PasswordPolicyControl

### PR DESCRIPTION
**Pull request by its-sami**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@ldapjs/messages",
   "homepage": "https://github.com/ldapjs/messages",
   "description": "API for creating and parsing LDAP messages",
-  "version": "1.2.1",
+  "version": "1.3.0",
```
Comment (jsumners) [inline package.json:5]: Please don't change the version number. I'll take care of this when I publish the module.

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "@ldapjs/asn1": "2.0.0",
     "@ldapjs/attribute": "1.0.0",
     "@ldapjs/change": "1.0.0",
-    "@ldapjs/controls": "2.0.0",
+    "@ldapjs/controls": "2.1.0",
```
Comment (jsumners) [inline package.json:16]: Ugh, some tool I used removed the `^` qualifier. Would you mind adding it to all of the dependencies as part of this PR?

```suggestion
    "@ldapjs/controls": "^2.1.0",
```

**Review by jsumners — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (its-sami) [reply package.json]: Cool I'll revert

File: package.json (hunk 2 above)
Comment (its-sami) [reply package.json]: Sure, of course

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Pull request merged**